pyvera/__init__.py

```diff
@@ -5,9 +5,24 @@
 """
 import logging
 import requests
+import sys
 
 from .subscribe import SubscriptionRegistry
 
+
+# activate to see raw http requests
+debug=0
+if debug:
+    import httplib as http_client
+    http_client.HTTPConnection.debuglevel = 1
+    logging.basicConfig()
+    logging.getLogger().setLevel(logging.DEBUG)
+    requests_log = logging.getLogger("requests.packages.urllib3")
+    requests_log.setLevel(logging.DEBUG)
+    requests_log.propagate = True
+
```

Comment:
Thanks for the PR - look like a few extra blank lines made it into your debug logic.

I'm slowly trying to improve the code to get through pep8 / pylint - so would you mind checking the new code gets through OK.

Then happy to merge.

Reply:
Oh, forgive me, i'm a bit new to this with pull request. I thought it was tied to the latest commit when issuing the request, but i see now that it takes the master branch as it updates. I'll clean out the code i wrote after the request and make it nice and tidy.